tests/integration/update.bats

```diff
@@ -48,6 +48,7 @@ function check_cgroup_value() {
 }
 
 @test "update" {
+    requires cgroups_kmem
```

Comment:
Maybe we should split this test into a version that just tests kernel memory updating? I don't like skipping all of the `update` code if you're on an older kernel.